src/components/Dropdown.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, FormEvent, OptionHTMLAttributes, ReactElement, createElement } from "react";
+import { CommonProps } from "../utils/ContainerUtils";
+
+export interface DropdownProps extends CommonProps {
+    onDropdownChangeAction?: (attribute: string, order: string) => void;
+    style: object;
+}
+
+export interface DropdownState {
+    attribute: string;
+    order: string;
+}
+
+interface DropdownType extends OptionHTMLAttributes<HTMLOptionElement> {
+    order: string;
+}
+
+export class Dropdown extends Component<DropdownProps, DropdownState> {
+
+    constructor(props: DropdownProps) {
+        super(props);
+
+        this.state = { attribute: "", order: "asc" };
+        this.updateSort = this.updateSort.bind(this);
+        this.resetQuery = this.resetQuery.bind(this);
+    }
+
+    render() {
+        return createElement("div", { className: "dropdown" },
```

Comment:
use `widget-<widgetname>` class name

Reply:
i feel this is not necessary given that the component is a child of the main widget and the main widget has already its class name `widget-listview-sort`